.github/ISSUE_TEMPLATE/accessibility.yml

```diff
@@ -112,11 +114,16 @@ body:
         - ArcGIS Monitor
         - ArcGIS Network Analyst
         - ArcGIS Online
+        - ArcGIS Pro
+        - ArcGIS SDK for JavaScript
```

Comment:
I think this and similar entries should be: [ArcGIS Maps SDK for JavaScript](https://developers.arcgis.com/javascript/latest/).

Reply:
Which ones are similar entries?